frontend/test/metabase/scenarios/native-filters/reproductions/12581.cy.spec.js

```diff
@@ -46,23 +48,33 @@ describe("issue 12581", () => {
       .type("{selectall}{backspace}", { delay: 50 });
     cy.get("@editor")
       .click()
-      .type("{selectall}{backspace}SELECT * FROM ORDERS");
+      .type("{selectall}{backspace}SELECT 1");
```

Comment:
This had nothing to do with the flake.
But it was wasteful to run the load the full "Orders" table. The query really doesn't matter at this step. What matters is that we create a new history point.